### ISSUE TITLE
Sentence Advisory Techniques for 2.4.7: Focus Visible need adjustments

### ISSUE THREAD

**Issue by jake-abma** [w3c/wcag#729]
!!! This comment may apply to more SC where the "(future link) sentences" are removed.

In 2.1 we've removed the superfluous possible techniques with "future link" at the end.
What we didn't do yet is adjusting the sentences preceding these possible technique.

The result is that we have sentences talking about what you can expect next, but there's nothing next. This is confusing and ready for an update too.

https://www.w3.org/WAI/WCAG21/Understanding/focus-visible.html#techniques 

> Advisory Techniques
> Although not required for conformance, the following additional techniques should be considered in order to make content more accessible. Not all techniques can be used or would be effective in all situations.
> ... [nothing following]...

Also here we might add the same solution for "Related resources" and add "none currently documented"

This one is closely related to: https://github.com/w3c/wcag/issues/705

**Comment by jake-abma** [comment]
When WCAG repeatedly have a lot of "related resources"and "Advisory techniques" contain "empty sentences" and do not have any valuable content it might even better to remove them instead of bothering people with extra content telling them they are "none currently documented"

**Comment by jake-abma** [comment]
These issues are not present anymore, great!
